src/server/index.js

```diff
@@ -27,25 +27,11 @@ app.use(cors());
 app.use(express.json())
 
 app.get('/api/pokedex', (req, res) => {
-    const promises = [];
-    for (let i = 1; i <= 150; i++) {
-        const url = `https://pokeapi.co/api/v2/pokemon/${i}`;
-        promises.push(fetch(url).then((res) => res.json()));
-    }
-    Promise.all(promises).then((results) => {
-        const pokemons = results.map((result) => {
-            let pokemon = {
-                name: result.name,
-                image: result.sprites['front_default'],
-                type: result.types.map((type) => type.type.name).join(', '), // if remove .join() then type = []
-                id: result.id
-            }
-            // UNCOMMENT NEXT LINE FOR SEED DATA 
-            // new PokemonModel(pokemon).save().catch(e => console.log(e))
-            return pokemon
-        })
-        res.json(pokemons)
-    });
+    PokemonModel.find().lean()
```

Comment:
what does  .lean() method do?

Reply:
model.find returns a large mongo document.. adding lean to the chain cuz away unneeded data and simplifies it just the obj at hand.. this code will still work the same way without .lean() maybe just a tad bit slower